src/target/move.c

```diff
@@ -320,7 +320,6 @@ void move_front_many(int cells)
  */
 void parametric_move_front(float distance, float end_linear_speed)
 {
-	side_sensors_control(true);
```

Comment:
Now we can move front in diagonals too. In diagonals, we do not want to activate side sensors control, so we are now simply assuming the walls-control activation is done somewhere else, but outside this function.

We cannot disable always the side sensors control here either, because we may want to activate it for straight (not diagonal) front movements.

Reply:
Maybe we can add in the speed/move refactoring issue a comment to state that this method could (should?) be named `parametric_move_straight` instead. To differentiate between straight-front and straight-diagonal. I do not know which is the best name, but we can consider a change when refactoring. :wink:

Reply:
I like to include it that on an issue to do not forget. #352 updated